{"name": "test_rerun", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 2", "trace": "def test_rerun():\n>       assert 1 == 2\nE       assert 1 == 2\n\ntest_rerun.py:9: AssertionError"}, "start": 1612242290336, "stop": 1612242290336, "uuid": "f719baf8-0ca4-473a-aa8e-1675d0c1fa7a", "historyId": "47c35f179f4b9f2c17e761a436dbbee0", "testCaseId": "558bc7562212911663d26c2ca57bd815", "fullName": "test_rerun#test_rerun", "labels": [{"name": "suite", "value": "test_rerun"}, {"name": "host", "value": "wangwei"}, {"name": "thread", "value": "1056-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_rerun"}]}